onnxruntime/test/providers/cpu/tensor/gather_elements_op_test.cc

```diff
@@ -158,7 +161,10 @@ void RunTypedTest<std::string>() {
   test4.AddOutput<std::string>("output", {2, 2},
                                {"a", "a",
                                 "d", "d"});
-  test4.Run(OpTester::ExpectResult::kExpectFailure, "GatherElements op: Value in indices must be within bounds [-2 , 1]. Actual value is -3");
+  // skip nuphar, which will not throw error message but will ensure no out-of-bound access
```

Comment:
>which will not throw error message but will ensure no out-of-bound access [](start = 18, length = 73)

I believe this should be the default behavior in ONNX spec for this op. Lots of devices won't be able to throw on invalid indices. Please open an issue to ONNX.

Reply:
OK